pocs/mount/__init__.py

```diff
@@ -1 +1,77 @@
+from glob import glob
+
 from pocs.mount.mount import AbstractMount  # pragma: no flakes
+from pocs.utils import error
+from pocs.utils import load_module
+from pocs.utils.error import MountNotFound
+from pocs.utils.location import create_location_from_config
+from pocs.utils.logger import get_root_logger
+
+
+def create_mount_from_config(config, mount_info=None, earth_location=None, *args, **kwargs):
+    """Create a mount instance based on the provided config.
+
+    Creates an instance of the AbstractMount sub-class in the module specified in the config.
+    Specifically, the class must be in a file called pocs/mount/<DRIVER_NAME>.py,
+    and the class must be called Mount.
+
+    Args:
+        config: A dictionary of name to value, as produced by panoptes.utils.config.load_config.
+        mount_info: Optional param which overrides the 'mount' entry in config if provided.
+            Useful for testing.
+        earth_location: astropy.coordinates.EarthLocation instance, representing the
```

Comment:
```suggestion
        earth_location: `astropy.coordinates.EarthLocation` instance, representing the
```

Same thing regarding documentation.